.internal-ci/util/wait-for-full-service.sh

```diff
@@ -4,22 +4,36 @@ set -e
 set -o pipefail
 
 
-echo "Checking block height - wait for full-service to start"
-sleep 15
+echo "-- Checking block height - wait for full-service to start"
+
+TARGET_FS_URL=${TARGET_FS_URL:-"http://localhost:9090/wallet/v2"}
 
 curl_post()
 {
-    curl --connect-timeout 2 -sSL -X POST -H 'Content-type: application/json' http://localhost:9090/wallet/v2 --data '{"method": "get_wallet_status", "jsonrpc": "2.0", "id": 1}' 2>/dev/null
+    curl --connect-timeout 2 -sSfL -X POST -H 'Content-type: application/json' "${TARGET_FS_URL}" --data '{"method": "get_wallet_status", "jsonrpc": "2.0", "id": 1}' 2>/dev/null
```

Comment:
`get_network_status` will return the `network_block_height` and `local_block_height` often in a fraction of the time as `get_wallet_status`, which is also calculating the status of all the txos of all the accounts in the wallet.